client/src/reviewList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+function ReviewList(props) {
+  console.log('props', props)
+      return (
+        <div id='container'>
+
+          {props.list.map((item) => {
+            return (
+            <div className='review-item'>
+              <div className='userName'>{item.userName}</div>
+              <img className='userUrl' src={item.imgUrl}/>
+              <div className='userReview'>{item.review}</div>
+            </div>
+
+              )
+          })}
```

Comment:
fun tip, if you use the arrow function without the { } braces, you dont need to say return, so for this map, you could jsut say : `props.list.map((item) => (
            <div className='review-item'>
              <div className='userName'>{item.userName}</div>
              <img className='userUrl' src={item.imgUrl}/>
              <div className='userReview'>{item.review}</div>
            </div>
              )
          )`